scrapers/TaraTainton.yml

```
@@ -0,0 +1,26 @@
+name: TaraTainton.com
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.taratainton.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@id="leftColumn"]/article/h1
+      Details:
+        selector: //div[@id="leftColumn"]/article/p
+        concat: "\n\n"
+      Studio:
+        Name:
+            fixed: Tara Tainton
+      Date: 
+        selector: //div[@id="leftColumn"]/article/div[@class="theDate"]
+        postProcess:
+          - parseDate: 02 January 2006
+      Tags:
+        Name: //div[@id="leftColumn"]/article/div[@class="tags"]/a
+      #Image: //div[@id="leftColumn"]/article/div[@clas="videoShow"]/hana-ampersand[2]/img
```

```suggestion
      Image: //meta[@property="og:image"]/@content
```
From a quick look you can get the scene image that is displayed in the main/index page